Iterable-iOS-SDK/IterableAlertViewController.m

```diff
@@ -369,17 +366,14 @@ -(void)ITESetData:(NSDictionary *)jsonPayload {
                 NSString *title;
                 if ([button objectForKey:ITERABLE_IN_APP_CONTENT]) {
                     NSDictionary* buttonContent = [button objectForKey:ITERABLE_IN_APP_CONTENT];
-                    if ([buttonContent objectForKey:ITERABLE_IN_APP_TEXT_FONT]) {
-                        self.buttonTitleFont = [UIFont fontWithName:[buttonContent objectForKey:ITERABLE_IN_APP_TEXT_FONT] size:self.buttonTitleFont.pointSize];
-                    }
                     if ([buttonContent objectForKey:ITERABLE_IN_APP_TEXT_COLOR]) {
                         self.buttonTitleColor = UIColorFromRGB([IterableInAppManager getIntColorFromKey:buttonContent keyString:ITERABLE_IN_APP_TEXT_COLOR]);
                     }
                     title = [buttonContent objectForKey:ITERABLE_IN_APP_TEXT];
                 }
                 self.buttonColor = UIColorFromRGB([IterableInAppManager getIntColorFromKey:button keyString:ITERABLE_IN_APP_BACKGROUND_COLOR]);
                 
-                [self addAction:[IterableAlertAction actionWithTitle:NSLocalizedString([button objectForKey:ITERABLE_IN_APP_TEXT], nil)
+                [self addAction:[IterableAlertAction actionWithTitle:NSLocalizedString(title, nil)
```

Comment:
Does this add fonts or what was the issue?

Reply:
It doesn't parse the title properly so it comes back at empty.

Reply:
It should have been [buttonContent objectForKey:ITERABLE_IN_APP_TEXT].

Reply:
Hm so why is it title now instead? Haha maybe you can just merge it if you think it fixes it since I it might take a while for me to get context on the mobile end and I wouldn’t even be able to test it.

Reply:
Hm so why is it title now instead? Haha maybe you can just merge it if you think it fixes it since I it might take a while for me to get context on the mobile end and I wouldn’t even be able to test it.

Reply:
It just uses the variable defined above on line 372
 title = [buttonContent objectForKey:ITERABLE_IN_APP_TEXT];